lib/firecord/configuration.rb

```diff
@@ -0,0 +1,27 @@
+module Firecord
+  module Configuration
+    CREDENTIALS_FILE = './credentials.json'.freeze
+
+    VALID_ACCESSORS = [:credentials_file]
+
+    attr_accessor *VALID_ACCESSORS
```

Comment:
Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication.